tests/datatypes/test_str.py

```diff
@@ -4,7 +4,6 @@
 
 
 class StrTests(TranspileTestCase):
-    @unittest.expectedFailure
```

Comment:
I think this test need to be updated? It reads that you _should_ be able to set arbitrary attrs on a `String` object. I'm not sure that's the case, but either the test or your code needs updating.

Thanks for the PR!